lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -834,6 +834,13 @@ namespace {
         }
       }
 
+      // The children of #selector expressions are not evaluation, so we do not
```

Comment:
s/are not evaluation/are not evaluated ?

Reply:
Thanks, I'll pick that up in my next PR, or this one if CI fails for some reason.